### ISSUE TITLE
No support for geometry data type field?

### ISSUE THREAD

**Issue by virtser** [ForestAdmin/lumber#51]
I have a MySQL database with table which has the following field:
``geolocation: DataTypes.GEOMETRY('POINT')``

While looking into that filed with simple MySQL client it has the following value:
``POINT(34.7787543 32.0708966)``

But Lumber/Forest shows this field as empty value, which makes it useless to update or display geo location widget. 😢 

![forest_ _instant__customizable_admin_interface_](https://cloud.githubusercontent.com/assets/468590/21524538/78eeb522-cd1f-11e6-958c-413774ace7b9.png)

**Comment by SeyZ** [comment]
Hey @virtser What do you want to see as an GEOMETRY type field on Forest? A Google map? Foretst supports "widgets" and definitively it makes sense to render correctly these type of fields.

**Comment by virtser** [comment]
Yes, definitely a Google map widget and allow to change position and save
it back to database.

**Comment by nickevansuk** [comment]
+1 for this, and even better if you can move the pin on the map to "correct" the lat lng
